src/garage/envs/__init__.py

```diff
@@ -4,11 +4,24 @@
 from garage.envs.base import Step
 from garage.envs.env_spec import EnvSpec
 from garage.envs.grid_world_env import GridWorldEnv
+from garage.envs.multi_env_wrapper import (MultiEnvWrapper,
+                                           round_robin_strategy,
```

Comment:
you shouldn't import round_robin_strategy, etc. into this package space -- only MultiEnvWrapper